### DISCUSSION TITLE
翻译与评述《后开放零成本许可证》

### DISCUSSION THREAD

**Discussion by giscus** [zhuangbiaowei/zhuangbiaowei.github.io#21] [Announcements]
# 翻译与评述《后开放零成本许可证》

特别感谢：赵云虎律师帮我审阅了翻译的初稿，并提出了很多专业的修改意见，详情请参见commit diff

https://zhuangbiaowei.github.io/2024/03/12/post-open-zero-cost-license.html

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
WoW, 这几乎是将一系列关键词, 在新时代重新进行了固化和说明;
可以拿来作日常沟通的基础了...
这可能也是历史上首个针对普通人代码权益的草案了?

虽然核心问题没有解决...法院是否愿意使用该草案,
但是, 好在给了一个明确的起点, 可以让所有普通开发者一起来理解并设计维护自身应有权益的正确姿势...

俺的直觉问题在:
为什么这类草案不是中国首先提出的?
比如华为, 一直在吃相关的苦, 却从来没有想过用许可证之类共同公约形式来确立基本权益呢?

以及神奇的是:"The POST-OPEN ADMINISTRATION" 后开放管理机构的提出,
这是将 GNU 设计的用版权法的原理, 构造出一个对等的自动发生的保护机制,
迁移为一个全球性的, 类似联合国的专门机构来代表全体开发者, 和各种混乱集会体一般的大厂进行谈判...

这难道不是倒退嘛?
谁又能来保证这个 The 开头的机构永远站在人民一边?
